docs/develop/local_dev.mdx

```diff
@@ -41,43 +42,47 @@ Restate is a single self-contained binary. No external dependencies needed.
             BIN=$HOME/.local/bin && RESTATE_PLATFORM=x86_64-unknown-linux-musl && \
             curl -LO https://github.com/restatedev/restate/releases/latest/download/restate.$RESTATE_PLATFORM.tar.gz && \
```

Comment:
@jackkleeman should we update this page to also use the Scarf links? is there an equivalent to the `latest` tag there?

Reply:
i think these links will stop working with the next release as the filename format is different. leave it in this PR and ill do a big effort to fix all the docs

Reply:
Let's merge this for now, I'll create a follow-up issue.